.github/workflows/build.yaml

```diff
@@ -95,6 +114,50 @@ jobs:
           name: debian-packages
           path: src/packages/build/ubuntu22.04/*.deb
           compression-level: 0 #No point in compressing these
+  BuildAndPackageWithUnitTestsOnArm:
+    name: Build, test and package code for arm architecture
+    needs: BuildAndPackageWithUnitTests
+    runs-on: ubuntu-22.04-arm
+    steps:
+      - name: Ensure required packages
+        env:
+          DEBIAN_FRONTEND: noninteractive # Less output to log
+        run: sudo apt-get update && sudo apt-get install -y curl software-properties-common build-essential unzip debhelper devscripts
+      - name: Set up JDK 21
+        uses: actions/setup-java@v4
+        with:
+          java-version: '21'
+          distribution: 'temurin'
+      - name: Set up Gradle
+        uses: gradle/actions/setup-gradle@v4
+        with:
+          cache-read-only: ${{ github.ref != 'refs/heads/develop' }}
+          gradle-home-cache-cleanup: true
+          dependency-graph: generate-and-submit
+          add-job-summary-as-pr-comment: always
+      - name: Download workspace
+        uses: pyTooling/download-artifact@v4 # https://github.com/actions/upload-artifact/issues/38
+        with:
+          name: workspace-for-arm
+      - name: Build native dependencies
+        working-directory: ./src
+        run: ./gradlew :service:signer:signer-api:make
+      - name: Build 22.04 (Jammy) packages
```

Comment:
Should we consider building RHEL packages on ARM as well?
They're not needed elsewhere in the workflow, but it could serve as an additional check to verify that RHEL builds work on ARM.